lib/client.js

```diff
@@ -283,6 +283,7 @@ function isOriginValid (origin) {
       /^https:\/\/.+\.zd-dev\.com$/,
       /^https:\/\/.+\.zd-master\.com$/,
       /^https:\/\/.+\.zendesk-staging\.com$/,
+      /^https:\/\/.+\.zendesk-dev\.com$/,
```

Comment:
we can further decrease the scope to specifc region. Currently there are 2 regions 

`.euc1.scooter.zendesk-dev.com`
`.usw2.scooter.zendesk-dev.com`

and a third in apac is on its way. https://zendesk.atlassian.net/browse/PLAN-7053

Reply:
Domain level looks fine to me